schemas/2018-08-20-preview/Microsoft.HealthcareApis.json

```diff
@@ -171,115 +137,100 @@
       "type": "object",
       "properties": {
         "origins": {
-          "readOnly": false,
           "oneOf": [
             {
               "type": "array",
-              "description": "The origins to be allowed via CORS.",
               "items": {
-                "anyOf": [
-                  {
-                    "type": "string",
-                    "description": "An origin for CORS.",
-                    "pattern": "^(?:(?:(?:[hH][tT][tT][pP](?:[sS]|))\\:\\/\\/(?:[a-zA-Z0-9-]+[.]?)+(?:\\:[0-9]{1,5})?|[*]))$"
-                  },
-                  {
-                    "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-                  }
-                ]
+                "type": "string"
               }
             },
             {
               "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
             }
-          ]
+          ],
+          "description": "The origins to be allowed via CORS."
         },
         "headers": {
-          "readOnly": false,
           "oneOf": [
             {
               "type": "array",
-              "description": "The headers to be allowed via CORS.",
               "items": {
-                "anyOf": [
-                  {
-                    "type": "string",
-                    "description": "A header for CORS."
-                  },
-                  {
-                    "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-                  }
```

Comment:
Is there a reason why its been removed?

Reply:
Yes - I think the generator is skipping the "expression" because it's a more restrictive form of string. i.e. there's no point in saying "property X is either any string, or it is an expression string", since that's logically equivalent to saying "property X is any string".